languages/php/src/BitwardenSDK.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Bitwarden\Sdk;
+
+use Bitwarden\Sdk\Schemas\AccessTokenLoginRequest;
+use Bitwarden\Sdk\schemas\ClientSettings;
+use Bitwarden\Sdk\Schemas\Command;
+use FFI;
+use Swaggest\JsonDiff\Exception;
+
+
+class BitwardenSDK
+{
+    private BitwardenLib $bitwarden_lib;
+
+    private ClientSettings $clientSettings;
+
+    public ProjectsClient $projectsClient;
+
+    public SecretsClient $secretsClient;
```

Comment:
Can we create functions to return these to keep the API similar to the others?
```
$bitwarden_client->secrets()->list(...)
```

Reply:
renamed properties, ie, secrets and projects used.